ntc_templates/templates/index

```diff
@@ -282,9 +282,11 @@ cisco_ios_show_vlan.textfsm, .*, cisco_ios, sh[[ow]] vlan
 cisco_ios_show_vrf.textfsm, .*, cisco_ios, sh[[ow]] vrf
 cisco_ios_dir.textfsm,  .*, cisco_ios, dir
 
+
```

Comment:
Small nit, can we remove the added new line here.
```suggestion
```

Reply:
VS Code must have automagically added that - sorry about that!